polyfill/spatnav-heuristic.js

```diff
@@ -441,15 +413,41 @@ function focusNavigationHeuristics() {
           focusables.push(thisElement);
         }
         else {
-          let recursiveFocusables = focusableAreas(thisElement);
+          const recursiveFocusables = thisElement.focusableAreas();
 
           if(Array.isArray(recursiveFocusables) && recursiveFocusables.length){
             focusables = focusables.concat(recursiveFocusables);
           }
         }
       }
     }
-    return focusables;
+
+    if (option === 'all') {
```

Comment:
`option` should not directly be a string equal to `"all"` or `"visible"`, but an object with a `mode` member, and `mode` is a string equal to `"all"` or `"visible"`.

Reply:
Thanks for the comment! I've fixed this.